manifest/fluentbit-to-fluentd-mode/output-to-different-kafka-topics-with-fluentd/fluentbit-config.yaml

```diff
@@ -0,0 +1,21 @@
+apiVersion: fluentbit.fluent.io/v1alpha2
+kind: ClusterFluentBitConfig
```

Comment:
We needn't add the same fluentbit stuff into every subfolder of `fluentbit-to-fluentd-mode` repeatedly.
And the same is for the file fluentbit-forward.yaml